packages/regl-worldview/src/Worldview.js

```diff
@@ -196,8 +196,12 @@ export class WorldviewBase extends React.Component<BaseProps, State> {
       return;
     }
 
-    if (!readHitmap && worldviewHandler) {
-      return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
+    // rendering the hitmap on mouse move is expensive, so disable it by default
+    if (mouseEventName === "onMouseMove" && !this.props.hitmapOnMouseMove) {
+      if (worldviewHandler) {
+        return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
+      }
+      return;
```

Comment:
#nit for clearer logic path: 
```
if (!this.props.hitmapOnMouseMove) {
  if (mouseEventName === "onMouseMove") {
    return;
  }
  if (worldviewHandler) {
    return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
  }
}

Reply:
I don't think this is exactly the same logic. I could change it to:
```
    if (mouseEventName === "onMouseMove" && !this.props.hitmapOnMouseMove) {
      if (worldviewHandler) {
        return handleWorldviewMouseInteraction(0, ray, e, worldviewHandler);
      } else {
        return;
      }
    }
```
Would that help?

Reply:
think the original is fine, this `else` is extraneous